src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/GroovyCallSiteSelector.java

```diff
@@ -193,15 +193,17 @@ private static boolean isInstancePrimitive(@Nonnull Class<?> type, @Nonnull Obje
         return findMatchingMethod(receiver, method, args);
     }
 
-    private static Method findMatchingMethod(Class<?> receiver, String method, Object[] args) {
+    private static Method findMatchingMethod(@Nonnull Class<?> receiver, @Nonnull String method, @Nonnull Object[] args) {
         Method candidate = null;
 
         for (Method m : receiver.getDeclaredMethods()) {
-            boolean isVarArgs = isVarArgsMethod(m, args);
-            if (m.getName().equals(method) && (matches(m.getParameterTypes(), args, isVarArgs))) {
-                if (candidate == null || isMoreSpecific(m, m.getParameterTypes(), isVarArgs, candidate,
-                        candidate.getParameterTypes(), isVarArgsMethod(candidate, args))) {
-                    candidate = m;
+            if (m != null) {
```

Comment:
Huh? `getDeclaredMethods` cannot return an array containing nulls. I think you can revert this hunk.

Reply:
I got paranoid. Will clean up.